pbs/README.md

```diff
@@ -0,0 +1,57 @@
+# PBS
+
+Proposer builder seperation for Op Stack.
+
+## How to run the devnet
+
+You first need to build the op-geth docker image with builder API support.
+
+```shell
+$ git clone git@github.com:flashbots/op-geth.git
+$ cd op-geth
+$ docker build . -t <YOUR_OP_GETH_BUILDER_IMAGE>
+```
```

Comment:
I think this can be in a different section for users who want to use their custom op-geth node